jujugui/static/gui/src/app/init.js

```diff
@@ -1619,7 +1620,7 @@ class GUIApp {
     this.db.destroy();
     this.endpointsController.destroy();
     // Detach event listeners.
-    const remove = document.removeEventListener;
+    const remove = document.removeEventListener.bind(document);
```

Comment:
Without this, the `remove()` calls no longer have access to the proper `document` context so it never removes the events causing cascading test failures